smoketest/graphqlalert_test.go

```diff
@@ -87,114 +94,190 @@ func TestGraphQLAlert(t *testing.T) {
 
 	doQL(fmt.Sprintf(`
 		mutation {
-			createNotificationRule(input:{
-				user_id: "%s"
-				contact_method_id: "%s",
-				delay_minutes: 0
+			sendContactMethodVerification(input:{
+				contactMethodID: "%s"
+			})
+		}
+	`, cm1.CreateUserContactMethod.ID), nil)
+
+	doQL(fmt.Sprintf(`
+		mutation {
+			sendContactMethodVerification(input:{
+				contactMethodID: "%s"
+			})
+		}
+	`, cm2.CreateUserContactMethod.ID), nil)
+
+	tw := h.Twilio()
+	d1 := tw.Device(phone1)
+	d2 := tw.Device(phone2)
+
+	msg1 := d1.ExpectSMS("verification")
+	msg2 := d2.ExpectSMS("verification")
+	tw.WaitAndAssert() // wait for code, and ensure no notifications went out
+
+	codeStr1 := strings.Map(func(r rune) rune {
```

Comment:
Probably just break out the map func instead of copying it, since they are used right here

```go
digits := func(r rune) rune {
    // ...
}
codeStr1 := strings.Map(digits, msg1.Body())
codeStr2 := strings.Map(digits, msg2.Body())
```